iocore/net/P_SSLNetVConnection.h

```diff
@@ -260,6 +260,17 @@ class SSLNetVConnection : public UnixNetVConnection
   /// Set by asynchronous hooks to request a specific operation.
   SslVConnOp hookOpRequested;
 
+  bool
+  getClientVerifyEnable()
+  {
+    return clientVerifyEnable;
+  }
+  void
+  setClientVerifyEnable(int enable)
```

Comment:
I think you should make this bool enable, and deal with that when you call. That seems more inline with the getter.